src/Response.php

```diff
@@ -93,7 +93,7 @@ public function addProperties(array $props)
      */
     public function __toString()
     {
-        if ($this instanceof ErrorResponse) {
+        if (get_class($this) !== "Zumba\VanillaJsConnect\Response") {
```

Comment:
@j-feng13 Rather than checking for this condition, just override the `__toString` method in the child classes.